src/mcts/node.cc

```diff
@@ -215,10 +232,67 @@ std::string Node::DebugString() const {
       << " WL:" << wl_ << " N:" << n_ << " N_:" << n_in_flight_
       << " Edges:" << static_cast<int>(num_edges_)
       << " Bounds:" << static_cast<int>(lower_bound_) - 2 << ","
-      << static_cast<int>(upper_bound_) - 2;
+      << static_cast<int>(upper_bound_) - 2
+      << " Solid:" << static_cast<bool>(solid_children_);
```

Comment:
nit: no need to cast as it's already a `bool`. (The others need cast from enum or avoiding being treated as `char`)

Reply:
done.